modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v2_0/test/DataDefinitionResourceTest.java

```diff
@@ -496,9 +496,9 @@ public void testPostDataDefinitionByContentType() throws Exception {
 		catch (Problem.ProblemException problemException) {
 			Problem problem = problemException.getProblem();
 
-			Assert.assertNull(problem.getDetail());
 			Assert.assertEquals("BAD_REQUEST", problem.getStatus());
 			Assert.assertEquals("MustSetValidType", problem.getType());
+			Assert.assertEquals("string", problem.getDetail());
```

Comment:
@marcelabc I would change this to the same order as the others, i.e., `problem.getDetail()` before `problem.getStatus()`